File: ui/src/main/java/com/midtrans/sdk/corekit/models/ShippingAddress.java
package com.midtrans.sdk.corekit.models;

public class ShippingAddress extends com.midtrans.sdk.uikit.api.model.Address {
    public ShippingAddress(String firstName, String lastName, String address, String city, String postalCode, String phone, String countryCode) {
        super(firstName, lastName, address, city, postalCode, phone, countryCode);
    }
}

File: corekit/src/main/kotlin/com/midtrans/sdk/corekit/api/model/ShippingAddress.kt
package com.midtrans.sdk.corekit.api.model

import com.google.gson.annotations.SerializedName
/**
 * It holds an information about shipping address of user.
 *
 * Created by shivam on 10/29/15.
 */
class ShippingAddress {
    var address: String? = null
    var city: String? = null
    var phone: String? = null

    @SerializedName("first_name")
    var firstName: String? = null

    @SerializedName("last_name")
    var lastName: String? = null

    @SerializedName("postal_code")
    var postalCode: String? = null

    @SerializedName("country_code")
    var countryCode: String? = null
}